app/views/shared/_form-actions.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="form-actions govuk-grid-row">
```

Comment:
Fair point, I'll revert to how it was before. I'll raise a separate PR to update this. The reason for adding this partial/component was because I noticed we were using the same thing in a few places and duplicating all of the stylings. Also, I don't agree with the use of a secondary button to cancel as it seems more like action rather than what it is (navigation back),

It seems like GOVUK Design system has brought it [button groups](https://design-system.service.gov.uk/components/button/#grouping-buttons) in the new version. Hopefully, we can just use that soon.